test/extended/etcd/helpers/helpers.go

```diff
@@ -354,6 +416,41 @@ func EnsureVotingMembersCount(ctx context.Context, t TestingT, etcdClientFactory
 	})
 }
 
+// WaitOnExpectedVotingMembersCount waits for 2 minutes and ensures the etcd memebership remains at the expected member count. Function returns a context deadline exceeded error if the
+// membership count stays at the expected member count and returns nil if the membership changes.
```

Comment:
> Function returns a context deadline exceeded error if the
// membership count stays at the expected member count and returns nil if the membership changes

This reads a bit confusing here and where it's invoked in the actual test. 
As the name `WaitOnExpectedVotingMembersCount ` suggests 
- The happy path is that the expected voting membership does not change
  - So we should return nil
- If the membership changes from the expected, that's unexpected
  -  So we return an error e.g something like `fmt.Errorf("failed to confirm that voting membersip should have remained constant, expected ...")`
  
 That way the test below doesn't also read like a double negative as you currently have it:
 > `o.Expect(err).To(o.HaveOccurred(), "voting member count shouldn't change from 3, as scale-down isn't expected to happen")`
 
 Instead more like
```
err = scalingtestinglibrary.WaitOnExpectedVotingMembersCount(ctx, g.GinkgoT(), etcdClientFactory, kubeClient, 3)
err = errors.Wrap(err, "scale-down should not have happened before scale up when cluster membership is healthy: ")
o.Expect(err).ToNot(o.HaveOccurred())
```